legacy_packages/sdk/src/evm/zksync/isZkContractDeployed.ts

```diff
@@ -0,0 +1,17 @@
+import { Provider } from "zksync-web3";
```

Comment:
side-note:

this lib is marked as deprecated, the new one supposedly is `zksync-ethers`, can we / should we upgrade as part of this work?

Reply:
see: https://github.com/zksync-sdk/zksync-ethers/tree/ethers-v5

Reply:
done